### ISSUE TITLE
Suggestions to add a new extensions for "listener-only nodes".

### ISSUE THREAD

**Issue by Inkeliz** [nanocurrency/raiblocks#1060]
The new version 15 includes a new message `bulk_pull_account`, allowing get pending blocks by account. It means that the RPC are not need to keep an _light wallet_. The _light wallet_ can talk with nodes directly, without a one dedicated server.

However, since theres _light wallet_ doesn't have all data, they can't be used to provide any usefull information for others nodes, the _light wallet_ maybe not re-broadcast messages too. It can create a pollution in the `keep_alive`, since it will include IPs of *light_wallets*, which is useless of others nodes.

My sugestion:

- Create a new "extesion" (in the header) to flag a "listener-only", nodes that are online just to get information, but not provide them. This type of node will not be include in the `keep_alive` for others.
- Create a new "reputation system", removing nodes that send the `keep_alive` but not do anything else.

**Comment by rkeene** [comment]
The tentative plan for this is to do it as a part of moving the real-time network to TCP and allow nodes to subscribe to events related to their address or participate fully in the network.  Nodes that subscribe to only their addresses will not be treated in the same way as full nodes.

**Comment by Inkeliz** [comment]
It's nice, also you reduce the ammount of data, but I'm not sure about the cost (for the node) to reply many nodes watching many single address.

However, I think that don't fixes the "zombie nodes". I didn't use the Raiblocks recently, my experience was in May/June. At this time many nodes didn't works in TCP, only UDP. Might be many reasons for that, which is not the point here. 

The problem is that kind of nodes (totally irresponsive, without TCP and unsynced) are given in `keep_alive`. When I tested the network multiples nodes didn't reply correctly, but still in the `keep_alive`.

Also, I think the keep_alive didn't follow the minimum version, the node can includes other nodes that are not compatible for the client. I don't know if this behavior was changed in new versions.
